hummingbird/ml/operator_converters/sklearn/imputer.py

```diff
@@ -28,10 +28,12 @@ def __init__(self, sklearn_imputer, device):
         self.replace_values = torch.nn.Parameter(
             torch.tensor([sklearn_imputer.statistics_], dtype=torch.float32), requires_grad=False
         )
-        self.missing_values = torch.nn.Parameter(
-            torch.tensor([sklearn_imputer.missing_values], dtype=torch.float32), requires_grad=False
-        )
+
         self.is_nan = True if (sklearn_imputer.missing_values == "NaN" or np.isnan(sklearn_imputer.missing_values)) else False
+        if not self.is_nan:
```

Comment:
Can you add a comment here about the bug requiring this order?

Reply:
I had to switch the 2 because when `self.is_nan` is true, `self.missing_values = torch.nn.Parameter(torch.tensor([sklearn_imputer.missing_values], dtype=torch.float32), requires_grad=False)` creates a tensor with `nan` as value which crashed PyTorch. (No idea how it was actually working before).